CONTRIBUTING_GO.md

```diff
@@ -161,6 +164,27 @@ reasons why this is _the right thing to do_:
 License: MIT
 Signed-off-by: Juan Benet <juan@ipfs.io>
 ```
+
+## Contributor Principles
+
+These principles are a living document and should be updated as-needed to better serve the go-ipfs community.
+
+In order of precedence:
+
+1. **Build the distributed web.** (may need a better way to capture this)
+2. **Don't break userspace.** This isn't "don't break protocols"; we do, in fact, occasionally break protocols (with care) when we know that users won't notice it.
+3. **Take ownership.** If you're working on something, own it and see it through. Given that (a) we don't have managers and (b) we don't work synchronously, coordination can be difficult. To make up for this, if you want something to happen, you have to make it happen (that may mean calling a meeting, bugging people, fixing unrelated issues, designing alternative solutions, etc.).
+4. **Ask for forgiveness, not permission.** To enable P.3, if you're having trouble getting people to move on something, err on the side of getting things done (bounded by P.1 and P.2). Note: This doesn't apply when you *know* you'll need to ask for forgiveness... use your best judgment.
+
+## Merge Policy
+
+The go merge policy must balance "not breaking things" with "getting things done" and should be read in that spirit.
+
+1. Except in special cases (and mass gx-workspace updates), all changes go through PRs.
+2. Small changes should get one review from someone who can sanity check the change.
```

Comment:
before being merged by whom?

Reply:
Hm. A merge policy that doesn't state who can merge what...

(the answer is: the PR author merges unless they're unsure, I'll update the doc).